client/js/components/procedure/SegmentsList/SegmentsList.vue

```diff
@@ -500,7 +500,7 @@ export default {
         .catch(() => {
           dplan.notify.notify('error', Translator.trans('error.generic'))
         })
-        .then(data => {
+        .finally(() => {
```

Comment:
sorry this was confusing from my side. I actually meant to add a `.finally()` after the `.then()` and set the `loadingState` to false in `finally`. We need the other implementation of `then()` and its logic and I think we can not simply replace the `then()` with a `finally()` here

Reply:
Ah, I see, I didn't get that. Done in [22216f9](https://github.com/demos-europe/demosplan-core/pull/1342/commits/22216f9cab3f6aae63a6e4c9667c1265c0f68fcf)